projects/web-app/src/components/app-shell/app-shell.vue

```diff
@@ -16,18 +18,35 @@
 </template>
 
 <script>
+import { ref, onUpdated, onErrorCaptured } from 'vue';
 import AppHeader from './app-header';
 import AppSidebar from './app-sidebar';
+import Loader from '../loader/loader';
```

Comment:
Should this component become a consumable?

Reply:
@bob-voorhoede Can it and should it be a composable?